### PR TITLE
Fix inherited "IsEnabled" property on Wasm

### PR DESCRIPTION
**Pull request by carldebilly**
GitHub Issue (If applicable): #1167 #1559

# Bugfix

## What is the current behavior?
`IsEnabled` is not correctly inherited on wasm.

## What is the new behavior?
On Wasm, setting `IsEnabled` on any control will also disable its children.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [X] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/SamplesApp/UITests.Shared/UITests.Shared.projitems [hunk 1]
```diff
@@ -4582,6 +4587,15 @@
...
     </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\Enability\BasicEnability.xaml.cs">
+      <DependentUpon>BasicEnability.xaml</DependentUpon>
+    </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\FocusManager\FocusManager_GetFocus_Automated.xaml.cs">
+      <DependentUpon>FocusManager_GetFocus_Automated.xaml</DependentUpon>
+    </Compile>
+    <Compile Update="C:\src\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml\IsEnabled\BasicEnability.xaml.cs">
+      <DependentUpon>BasicEnability.xaml</DependentUpon>
+    </Compile>
```
Comment (jeromelaban) [inline src/SamplesApp/UITests.Shared/UITests.Shared.projitems]: Remove :)

File: src/Uno.UI/UI/Xaml/Input/FocusManager.cs [hunk 2]
```diff
@@ -1,4 +1,6 @@
-﻿#if XAMARIN || __WASM__
+﻿
+using Windows.UI.Core;
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Input/FocusManager.cs]: In the block below?

File: src/SamplesApp/UITests.Shared/UITests.Shared.projitems (hunk 1 above)
Comment (carldebilly) [reply src/SamplesApp/UITests.Shared/UITests.Shared.projitems]: If I remove, we'll loose the file nesting in VS.  We need to move the <DependentUpon> at the right place in the file.  That's a lame VS is not able to do this.

File: src/Uno.UI/UI/Xaml/Input/FocusManager.cs (hunk 2 above)
Comment (carldebilly) [reply src/Uno.UI/UI/Xaml/Input/FocusManager.cs]: Ooh good catch.

File: src/SamplesApp/UITests.Shared/UITests.Shared.projitems (hunk 1 above)
Comment (MartinZikmund) [reply src/SamplesApp/UITests.Shared/UITests.Shared.projitems]: I think it is enough to just replace the absolute path with `$(MSBuildThisFileDirectory)` prefix. This makes it portable while retaining the nesting in VS

**Pull request merged**